### PR TITLE
Lab00 blink - Add pico blink and GitHub build workflow

### PR DESCRIPTION
**Pull request by muteebakram**

### CODE REVIEW

File: lab00-blink/README.md
Comment (akcase18) [inline lab00-blink/README.md:1]: Great job on this. Love the detail

**Review by akcase18 — APPROVED** [review]
Great job! No notes.

**Comment by TylerEvans-Dev** [comment]
I think you may have forgotten to add the GitHub badge to the readme page along with your names But other than that the YML file is checking and working. I hope this helps with the lab 0. Good work with what you have done thus far !

**Review by TylerEvans-Dev — APPROVED** [review]
everything looks good aside from the readme. Thanks hope this helps.

**Comment by muteebakram** [comment]
> everything looks good aside from the readme. Thanks hope this helps.

I know I tried fixing but it does not work for me for some reason. I will dig more and find out.

**Pull request merged**